cli/flox-rust-sdk/src/providers/build.rs

```diff
@@ -587,6 +587,8 @@ mod tests {
         let mut env = new_path_environment(&flox, &manifest);
         let env_path = env.parent_path().unwrap();
 
+        let _git = GitCommandProvider::init(&env_path, false).unwrap();
```

Comment:
Sandboxes test require the directory to be git tracked, this test failed because it did not setup a git dir.
It was expected to fail, but not like this.

We made the same change in the surrounding sandbox tests.